lib/db/schema/patch-070-071.sql

```diff
@@ -0,0 +1,81 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS totp (
+  uid BINARY(16) NOT NULL,
+  sharedSecret VARCHAR(80) NOT NULL,
+  epoch BIGINT NOT NULL,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  UNIQUE KEY (`uid`)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createTotpToken_1` (
+  IN `uidArg` BINARY(16),
+  IN `sharedSecretArg` VARCHAR(80),
+  IN `epochArg` BIGINT UNSIGNED,
+  IN `createdAtArg` BIGINT UNSIGNED
+)
+BEGIN
+
+  INSERT INTO totp(
+    uid,
+    sharedSecret,
+    epoch,
+    createdAt
+  )
+  VALUES(
+    uidArg,
+    sharedSecretArg,
+    epochArg,
+    createdAtArg
+  );
+
+END;
+
+CREATE PROCEDURE `getTotpToken_1` (
+  IN `uidArg` BINARY(16)
+)
+BEGIN
+
+  SELECT sharedSecret, epoch FROM totp WHERE uid = uidArg;
+
+END;
+
+CREATE PROCEDURE `deleteTotpToken_1` (
+  IN `uidArg` BINARY(16)
+)
+BEGIN
+
+  DELETE FROM totp WHERE uid = uidArg;
+
+END;
+
+CREATE PROCEDURE `deleteAccount_14` (
```

Comment:
Shouldn't we also delete from `totp` inside `resetAccount`?

Reply:
That detail for the account reset flow hasn't been finalized yet. I think it would be ok to add that bit in on another PR since there might be other changes for account reset.